GetStream/stream-chat-android
# [4042] Fallback UI for unsupported attachments in Compose

### 🎯 Goal

https://github.com/GetStream/stream-chat-android/issues/4042

### 🛠 Implementation details

Added fallback `UnsupportedAttachmentFactory` for attachments with unsupported types.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![photo_2022-10-12 11 06 28](https://user-images.githubusercontent.com/9600921/195286456-409dc7ed-88f1-4829-9dae-ee04c4118651.jpeg) | ![photo_2022-10-12 11 06 24](https://user-images.githubusercontent.com/9600921/195286463-9b3a122a-8eb4-410a-af6a-9ae3ab48be8f.jpeg) |

### 🧪 Testing

1. Open Compose sample app
2. Login as Dmitrii
3.  Open "unsupported attachments" channel
4. Ensure that unsupported attachments are displayed correctly

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist
